tests/phpunit/tests/blocks/editor.php

```diff
@@ -568,6 +568,41 @@ public function test_block_editor_rest_api_preload_adds_missing_leading_slash( a
 		$this->assertStringContainsString( $expected, $haystack );
 	}
 
+	/**
+	 * @ticket 57547
+	 */
+	public function test_get_classic_theme_supports_block_editor_settings() {
+		$font_sizes = array(
+			array(
+				'name' => 'Small',
+				'size' => 12,
+				'slug' => 'small',
+			),
+			array(
+				'name' => 'Regular',
+				'size' => 16,
+				'slug' => 'regular',
+			),
+		);
+
+		add_theme_support( 'editor-font-sizes', $font_sizes );
+		$settings = get_classic_theme_supports_block_editor_settings();
+		remove_theme_support( 'editor-font-sizes' );
+
+		$this->assertFalse( $settings['disableCustomColors'] );
+		$this->assertFalse( $settings['disableCustomFontSizes'] );
+		$this->assertFalse( $settings['disableCustomGradients'] );
+		$this->assertFalse( $settings['disableLayoutStyles'] );
+		$this->assertFalse( $settings['enableCustomLineHeight'] );
+		$this->assertFalse( $settings['enableCustomSpacing'] );
+		$this->assertFalse( $settings['enableCustomUnits'] );
+
+		$this->assertSame(
+			$font_sizes,
+			$settings['fontSizes']
+		);
```

Comment:
Please make use of the `$message` parameter for tests with multiple assertions, to explain what went wrong if one should fail (see [Using Assertions](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions) for details).